src/v2/Apps/Conversation/Components/OrderUpdate.tsx

```diff
@@ -55,16 +55,21 @@ export const OrderUpdate: React.FC<OrderUpdateProps> = ({
       return null
     }
   } else if (event.__typename === "CommerceOrderStateChangedEvent") {
-    const { orderUpdateState } = event
-    if (orderUpdateState === "offer_approved") {
+    const { orderUpdateState, state, stateReason } = event
+    const reasonLapsed = stateReason?.includes("_lapsed")
+    if (state === "APPROVED") {
       color = "green100"
-      message = `Offer Accepted`
+      message = `${
+        orderUpdateState === "offer_approved" ? "Offer" : "Purchase"
+      } Accepted`
     } else if (orderUpdateState === "offer_rejected") {
       color = "red100"
       message = `Offer Declined`
-    } else if (orderUpdateState === "offer_lapsed") {
-      color = "black60"
-      message = `Offer Expired`
+    } else if (state === "CANCELED" && reasonLapsed) {
```

Comment:
@tam-kis  If I understood correctly, we must add this use case to [order_status_helper](https://github.com/artsy/exchange/blob/main/lib/order_status_helper.rb), right?